course_discovery/services.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from edx_rest_api_client.client import EdxRestApiClient
+
+
+logger = logging.getLogger()
+
+
+class CatalogApiService(object):
```

Comment:
I was originally using the existing service and had my script at the top level of the repo. I couldn't have it in a sub directory because the top folder in the repo has a hyphen which breaks absolute python imports. 
Clinton suggested refactoring the whole repo, but since that would involve updating a bunch of jobs, he suggested alternatively to just make a new client